android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java

```diff
@@ -96,17 +100,21 @@ public void manageSubscriptions() {
                 "manageSubscriptions", Toast.LENGTH_SHORT
         ).show();
 
-         this.thisActivity = ((Activity) mWXSDKInstance.getContext());
-         Uri webpage = Uri.parse("http://play.google.com/store/account/subscriptions");
-         Intent intent = new Intent(Intent.ACTION_VIEW, webpage);
-         this.thisActivity.startActivity(intent);
+        this.thisActivity = ((Activity) mWXSDKInstance.getContext());
+        Uri webpage = Uri.parse("http://play.google.com/store/account/subscriptions");
+        Intent intent = new Intent(Intent.ACTION_VIEW, webpage);
+        this.thisActivity.startActivity(intent);
     }
 
     @JSMethod(uiThread = true)
     public void getProductInfo(String params, JSCallback jsCallback) {
         Log.d(TAG, "-> getProductInfo");
         JSONObject json = new JSONObject();
-        try { json = new JSONObject(params); } catch (Throwable t) { Log.e("-> show", "Could not parse malformed JSON: \"" + json + "\""); }
```

Comment:
@phanisaripalli is there any way we can refactor this to become a method and call it from both methods that are using this parsing?